test cases/common/262 configure file output format/meson.build

```diff
@@ -0,0 +1,37 @@
+project('configure file output format')
+
+data = configuration_data()
+data.set_quoted('str', 'hello world!', description: '''This is
```

Comment:
Replace this line with
```
data.set_quoted('quoted', 'hello world!', description: '''This is
a multiline
description''')
data.set('unquoted', 'float')
```
to increase the test coverage. (Then update expected/*)